tests/_regtest_outputs/test_run_topostats.test_process_scan_lower.out

```diff
@@ -1,3 +1,2 @@
-                  centre_x    centre_y  radius_min  radius_max  radius_mean  radius_median  height_min  height_max  height_median  height_mean     volume       area  area_cartesian_bbox  smallest_bounding_width  smallest_bounding_length  smallest_bounding_area  aspect_ratio threshold  max_feret  min_feret  Contour Lengths  Circular  End to End Distance      Image Name
-Molecule Number                                                                                                                                                                                                                                                                                                                                                                   
-0                312.31095  135.689968     2.04927    6.596742      4.42801       4.502005   -0.838749   -0.473677      -0.602617    -0.607314 -31.864748  52.468362           106.156919                 5.459192                  12.95475               70.722472      2.373016     lower  12.966989   5.459192        44.837437     False             5.948583  minicircle.spm
+   Molecule Number  centre_x  centre_y  radius_min  radius_max  radius_mean  radius_median  height_min  height_max  height_median  height_mean  volume  area  area_cartesian_bbox  smallest_bounding_width  smallest_bounding_length  smallest_bounding_area  aspect_ratio  Contour Lengths  Circular  End to End Distance  Image Name
+0              NaN       NaN       NaN         NaN         NaN          NaN            NaN         NaN         NaN            NaN          NaN     NaN   NaN                  NaN                      NaN                       NaN                     NaN           NaN              NaN       NaN                  NaN         NaN
```

Comment:
See above comment against `tests/_regtest_outputs/test_run_topostats.test_process_scan_both.out` about no longer having a test for lower scan threshold as nothing is detected.